CHANGELOG.txt

```diff
@@ -202,6 +208,14 @@
                 - "creme_core-hatmenubar-update"
                 - "creme_core-hatmenubar-view"
             - Apps :
+                * Activities:
+                    - Refactor creme.ActivityCalendar
+                        - Move the "calendar user page" features to creme.ActivityCalendarController : floating events, sidebar selection, state within url hash
+                        - creme.ActivityCalendar can be now used as independent component.
+                        - Add headlessMode flag.
+                        - Add allowEventOverlaps property (boolean or method).
+                        - Add allowEventCreate property (boolean).
+                        - Move calendar selection state queries to ActivityCalendarController. Improve performance.
```

Comment:
Are all these items breaking changes (because this is the section for breaking changes) ?